docs/userguide.md

```diff
@@ -232,6 +233,39 @@ NAME   SHORTNAMES   APIGROUP   NAMESPACED   KIND
 Verifying the API type exists on all member clusters will ensure successful
 propagation to that cluster.
 
+### Enabling an API type in a new federation group
+When `kubefedctl enable` is used to enable types whose plural names (e.g. **deployments**.example.com
+and **deployments**.apps) match, the crd name of the generated federated type would also match (e.g.
+**deployments**.types.federation.k8s.io).
+
+`kubefedctl enable --federation-group string` specifies the name of the API group to use for the
+generated federation type. It is `types.federation.k8s.io` by default. If a new federation group is
+enabled, the RBAC permissions for the federation controller manager will need to be updated to include
+permissions for the new group.
+
+For example, after federation deployment, `deployments.apps` is enabled by default. To enable
+`deployments.example.com`, you should:
+```bash
+kubefedctl enable deployments.example.com --federation-group example.com
```

Comment:
`s/--federation-group example.com/--federation-group federation.example.com/`

I don't think it would be desirable to reuse a non-federated group for federated resources because there is no guarantee that the non-federated resource is a CRD.  As per the following comment, mixing CRDs and non-CRDs in the same group is best avoided:

https://github.com/kubernetes/kubernetes/issues/68985#issuecomment-423790348